components/EmbedMedia/EmbedMedia.tsx

```diff
@@ -59,6 +71,13 @@ export const EmbedMedia = ({
               height="100%"
               url={mediaUrl}
               controls
+              playsinline
+              light={isPreview ? true : false}
+              playing={isPreview ? true : false}
+              playIcon={isPreview ? PlayPreviewIcon : undefined}
+              // This previewAriaLabel prop is not documented but it is in the React Player source code
+              previewAriaLabel={isPreview ? previewAriaLabel : undefined}
```

Comment:
SODA said this makes it ok accessible (it renders a clickable/focusable div). When I have time I'll open a PR to React Player to add a role="button" (SODA suggestion to make it fully accessible).

Reply:
You can always just log a request or file a bug. It is nice of you to submit the PR but sometimes you can just let others know if you don't have the time.